File: problem1.c
//solved
#include <stdio.h>

int main()
{
	int i, total = 0;
	for (i=1;i<1000;i++)
	{
		if ((i % 3 == 0) || (i % 5 == 0))
		{
			total += i;
			printf("Found multiple %d, total is %d\n", i, total);
		}
	}
	
	printf("The sum of all multiples of 3 or 5 below 1000 is: %d\n", total);
	
	return 0;
}

File: problem1.py
#!/usr/bin/python3.2

import sys

def main(argv):
	total = 0;
	for i in range(1,1000):
		if i % 3 == 0 or i % 5 == 0:
			print("Found multiple: " + str(i));
			total += i;
			
	print("The sum of all multiples of 3 and 5 from 1 to 1000 is " + str(total));
	return 0;

if __name__ == "__main__":
    main(sys.argv);
